vc-generator/secret.js

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ */
+// This is only used inside this test suite for generating vcs for the verify
+// proof tests.
+export const testDataGeneratorSeedMultibase =
```

Comment:
this should be an environment variable like TEST_KEY_SEED or something like that. We don't want to expose bs58 key seeds unless we have to, and yes I have the done same thing in the past.

Reply:
@aljones15 Updated to TEST_KEY_SEED - [81bc51c](https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/13/commits/81bc51ca3b71fa02500b3a421d686797e7d68677). I think this does not require to be secret since it is only used to generate test data here and not utilized by any implementations. Also, IMO setting it as an environment variable would introduce unnecessary complexity, as it would require passing an additional environment variable (TEST_KEY_SEED) when running other tests.

Reply:
I would prefer an ENV variable, but this is ok provided we reuse this test seed and don't expose more key seeds. Good work and feel free to merge when ready,